modules/xmpp/JingleSessionPC.js

```diff
@@ -543,7 +580,7 @@ export default class JingleSessionPC extends JingleSession {
     sendIceCandidate(candidate) {
         const localSDP = new SDP(this.peerconnection.localDescription.sdp);
 
-        if (candidate && !this.lasticecandidate) {
+        if (candidate && candidate.candidate !== '' && !this.lasticecandidate) {
```

Comment:
checkc the length > 0 maybe? I this what breaks FF74? Can we have it as a separate PR?

Reply:
I changed it but we are still not sure what the issue is with FF74 so I am leaving it here for now.